src/connect_wrap.cc

```diff
@@ -1,14 +1,18 @@
 #include "connect_wrap.h"
-
-#include "env-inl.h"
 #include "req_wrap-inl.h"
-#include "util-inl.h"
+
+namespace v8 {
+class Object;
+template <class T> class Local;
+}  // namespace v8
```

Comment:
This file includes v8.h indirectly anyway, should we add this here? (ditto below?)

Like, any file that actively uses V8 APIs will have it included anyway, through `node.h` or otherwise…

Reply:
I'll take a look, remove the declarations and let you know of the result (sometime tomorrow afternoon / evening)

Reply:
I think I've removed the declarations you pointed, and they were as you said not needed.

Were the ones removed by my last commit all the ones you were referring to??